src/index.css

```diff
@@ -170,4 +170,136 @@
     width: 3.125rem;
     height: 3.125rem;
   }
+
```

Comment:
If these styles are specifically only meant for a particular component (in this case, the file explorer table), we should 
put the stylesheet in the component's folder (and only imported by that particular component) to segregate these classes away from everything else.

For an instance of this, look at `/index.js` and `/index.css` in `src/components/Modal`